src/app/mock-list.ts

```diff
@@ -0,0 +1,14 @@
+import { List } from "./list";
+
+export const ITEMS : List[] = [
+  { id: 11, name: 'Dr Nice' },
```

Comment:
This isn't a list item. Look `ViewItemComponet` for the current definition of an item data